yarn.lock

```diff
@@ -14562,9 +14562,9 @@ http-assert@^1.3.0:
     http-errors "~1.8.0"
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26602&commitId=03bc67c6afe89a6dd704f3d22cde799eab7024e4&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Expandable.chromium.png|254| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>